internal/pkg/log/log.go

```diff
@@ -53,6 +53,11 @@ type baseLogger interface {
 }
 
 type contextLogger interface {
+	Debug(ctx context.Context, template string)
+	Info(ctx context.Context, template string)
+	Warn(ctx context.Context, template string)
+	Error(ctx context.Context, template string)
+
```

Comment:
Please add some comments here, how it works.

Reply:
Changes in the `logger` package can be a separate PR, but can be also here.

**But I am missing tests for the logger changes.**